a2lmacros/src/codegenerator/writer.rs

```diff
@@ -70,7 +70,7 @@ fn generate_block_writer_generic(typename: &str, structitems: &[DataItem]) -> To
 
     quote! {
         impl #typeident {
-            pub(crate) fn stringify(&self, indent: usize) -> String {
+            pub fn stringify(&self, indent: usize) -> String {
```

Comment:
If the stringify function is useful to you, then it could definitely be made public.
That should be a separate commit though, not part of the cleanup.

Reply:
oups! this is some debug leftovers. sorry!